packages/apm/src/integrations/tracing.ts

```diff
@@ -389,7 +414,7 @@ export class Tracing implements Integration {
    * Removes activity and finishes the span in case there is one
    */
   public static popActivity(id: number, spanData?: { [key: string]: any }): void {
-    if (!Tracing._isEnabled()) {
+    if (!Tracing._isEnabled() || !id) {
```

Comment:
```suggestion
    if (!Tracing._isEnabled() || id == null) {
```

or `typeof id === 'undefined'` since `!0` is truthy.

Reply:
This is on purpose, will add a code comment.

Reply:
`id <= 0` would also work 👍